org.opent2t.sample.binaryswitch.superpopular/com.smartthings.binaryswitch/js/thingTranslator.js

```diff
@@ -86,30 +86,7 @@ function findResource(schema, di, resourceId) {
     return resource;
 }
 
-function getDeviceResource(translator, di, resourceId) {
-    return translator.get(true)
-        .then(response => {
-            return findResource(response, di, resourceId);
-        });
-}
-
-function postDeviceResource(di, resourceId, payload) {
-    if (di === controlId){
-        var putPayload = resourceSchemaToProviderSchema(resourceId, payload);
-
-        return smartThingsHub.putDeviceDetailsAsync(controlId, putPayload)
-            .then((response) => {
-                var schema = providerSchemaToPlatformSchema(response, true);
-
-                return findResource(schema, di, resourceId);
-            });
-    } else {
-        throw new Error('NotFound');
-    }
-}
-
-var controlId;
-var smartThingsHub;
+const switchDeviceDi = "d455d979-d1f9-430a-8a15-61c432eda4a2";
```

Comment:
Lets talk about this in person with Avneet as I think our understanding of the deviceId may not be the same.

---
In reply to: [101098710](https://github.com/openT2T/translators/pull/160#discussion_r101098710) [](ancestors = 101098710)